### DISCUSSION TITLE
changing uyuni url

### DISCUSSION THREAD

**Discussion by fockarty** [uyuni-project/uyuni#9345] [Q&A]
Is there a wat to change the url of uyuni running in podman? I found a script spacewalk-hostname-rename which should do that but it isn't made for containerized installations so it failes on the ip address.

**Comment by sbluhm** [comment]
URL or hostname?
The scripts don't work fully. I managed to do it to 90% and would have to dig it out (I think cobbler still has the wrong name somewhere for the kickstart).

**Reply by fockarty** [reply]
only the url will do, Uyuni container uses hostname 'uyuni-server.mgr.internal' as default and the podman's host as url as in 'podman.company.nl'. I want the hostname to be 'uyuni.company.nl'

**Reply by sbluhm** [reply]
let me check later if I can find notes or so

**Reply by sbluhm** [reply]
@fockarty , unfortunately, I do not have my records anymore. Overall, I followed the steps of the spacewalk-hostname-rename script and and ran the commands inside with the correct values.

**Reply by fockarty** [reply]
ok, thanks, i already looked through the script, isn't too much work. The manuals/scripts aren't 100% container ready yet...

**Comment by fockarty** [comment]
ran through the renaming script and executed all commands by hand which worked perfectly :-)
